AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -112,6 +112,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+joinhoney.com##div[class^="cookieConsentBanner"]
```

Comment:
@gork7777 
Let's use` joinhoney.com##div[class^="cookieConsentBannerContainer"]` and please remove 2 obsolete rules